services/sso/src/main/java/software.amazon.awssdk.services.sso/auth/SsoCredentialsProvider.java

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.sso.auth;
```

Comment:
It looks like the directory name is `software.amazon.awssdk.services.sso` instead of having a chain of parent directories: `software/amazon/awssdk/services/sso`. Is that just github being weird, or is that accurate?

Reply:
It might be IntelliJ acting weird, I will recreate the directory to make it accurate.